src/recipes/lint.ts

```diff
@@ -24,6 +24,7 @@ const execute = async (
   // eslint@9 introduces new configuration format that is not supported by widely used plugins yet,
   // so we stick to ^8 for now.
   await toolbox.dependencies.addDev('eslint', context, { version: '^8' })
+  await toolbox.dependencies.addDev('typescript', context)
```

Comment:
What in the case when TypeScript isn't used in the project?

Reply:
Then we should not add react native extension to default `.eslintrc.json`. I created a task for this case and possibly other cases like expo or bare react: https://github.com/software-mansion-labs/react-native-ci-cli/issues/99